python/interpret_community/explanation/explanation.py

```diff
@@ -376,7 +376,8 @@ def is_local_sparse(self):
         :rtype: bool
         """
         local_vals = self._local_importance_values
-        return issparse(local_vals) or (isinstance(local_vals, list) and issparse(local_vals[0]))
+        return issparse(local_vals) or \
+            ((isinstance(local_vals, list) or isinstance(local_vals, np.ndarray)) and issparse(local_vals[0]))
```

Comment:
hmm this should always be a list of scipy sparse matrices or just a scipy sparse matrix, never an ndarray (of scipy sparse matrices or otherwise).  If it is an ndarray then something must be incorrect at an earlier point in the code, I would need to take a look.

Reply:
are you sure this change is required?

Reply:
In the testing, this function returned False because the raw explanations given were array of sparse matrices. That's why this check. Where earlier in the code execution do we have this format (list of sparse matrices) created?

Reply:
depends on the explainer, if it is linear explainer I believe it is in file linear_model in functions:
_compute_local_shap_values

```
    # Multiclass case
    if isinstance(linear_explainer, list):
        shap_values = []
        for explainer in linear_explainer:
            explainer_shap_values = explainer.shap_values(evaluation_examples)
            if isinstance(explainer_shap_values, list):
                explainer_shap_values = explainer_shap_values[0]
            shap_values.append(explainer_shap_values)
        return shap_values
```
we just append the sparse shap values to a list in multiclass case.  Hmm, I'd need to investigate more, but conceptually I always thought:
```
dense case, multiclass:
  multidimensional np.array
sparse case, multiclass:
  list of scipy sparse 2-D matrices
```
maybe I am getting confused somehow

Reply:
hmm, maybe let's merge this for now, and perhaps you can create a task for me to look into this later.  I'm not really sure what is going on here and this needs more debugging in the code.

Reply:
Sounds good. Not sure I am able to create a task in github. I will follow up with you offline. 

I will complete the PR.